src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -0,0 +1,325 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactTestRendererFiber
+ * @preventMunge
+ * @flow
+ */
+
+'use strict';
+
+var ReactFiberReconciler = require('ReactFiberReconciler');
+var ReactGenericBatching = require('ReactGenericBatching');
+var emptyObject = require('emptyObject');
+
+import type { TestRendererOptions } from 'ReactTestMount';
+
+type ReactTestRendererJSON = {
+  type: string,
+  props: { [propName: string]: string },
+  children: null | Array<string | number | ReactTestRendererJSON>,
+  $$typeof?: any
+}
+
+let instanceCounter = 0;
+
+class TestContainer {
+  createNodeMock: Function;
+
+  constructor(createNodeMock) {
+    this.createNodeMock = createNodeMock;
+  }
+  appendChild(child) {}
+  insertBefore(beforeChild, child) {}
+  removeChild(child) {}
+  toJSON() {}
```

Comment:
Maybe. I'm pretty sure them being noops is wrong (since they do get called), but I'm not sure how to write a failing test. If you succeed with making a failing test against this then you'll know how to fix it.